src/Tooltip/TooltipTouch.tsx

```diff
@@ -47,7 +61,7 @@ export default class TooltipTouch extends React.PureComponent {
     }, 200);
   }
 
-  onTouchMove = (e) => {
+  onTouchMove = (e: React.TouchEvent<HTMLDivElement>) => {
```

Comment:
minor, but you can annotate this as `onTouchMove: React.TouchEventHandler<HTMLDivElement>` and avoid the annotation on `e`